pyproject.toml

```diff
@@ -91,14 +91,6 @@ add_ignore = [
 [tool.pytest.ini_options]
 addopts = "--tb=native --strict-markers"
 testpaths = ["tests"]
-markers = [
-    "integration: tests that use a container (mostly databases)",
-    "spark: tests that use a Spark database (and therefore tend to be very slow)",
-]
 filterwarnings = [
     "ignore::DeprecationWarning:past.builtins.misc:45",
-    "ignore::DeprecationWarning:pytest_freezegun:17",
-    "ignore::DeprecationWarning:docker.utils.utils:52",
-    "ignore::DeprecationWarning:docker.utils.utils:53",
-    "ignore::DeprecationWarning:pyhive.common:248",
```

Comment:
This `pyhive` one is still needed, I think

Reply:
Yep, thanks.  I'll fix up once #909 is merged.